ovos_padatious/training_manager.py

```
@@ -72,37 +74,43 @@ def add(self, name: str, lines: List[str], reload_cache: bool = False, must_trai
+        hash_fn = join(self.cache, name + '.hash')
+        min_ver = splitext(ovos_padatious.__version__)[0]
+        if not isfile(hash_fn):
+            must_train = True
+
         if not must_train:
-            LOG.debug(f"Loading {name} from intent cache")
-            self.objects.append(self.cls.from_file(name=name, folder=self.cache))
+            try: # .net file renamed/deleted for some reason
+                LOG.debug(f"Loading '{name}' from intent cache")
+                self.objects.append(self.cls.from_file(name=name, folder=self.cache))
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)